include/openenclave/bits/result.h

```diff
@@ -314,6 +314,21 @@ typedef enum _oe_result
      */
     OE_VERIFY_FAILED_AES_CMAC_MISMATCH,
 
+    /**
+     * Failed to post a switchless call to host workers
```

Comment:
Don't use the term "switchless" by itself as it's ambiguous/confusing.  Say "context-switchless"